.gitignore

```diff
@@ -67,11 +41,3 @@ execution-engine/tests/*.lock
 execution-engine/gens/*.lock
 execution-engine/system-contracts/mint-token/*.lock
 execution-engine/system-contracts/pos/*.lock
-
-# wasms
```

Comment:
How about remain that one..?! This can be confusing when testing between casper sources.

What about using `.gitignore` as is?

Reply:
Do you mean the testing out execution-engine using CasperLabs's client and node?

If it is what you mean, we can test by cloning both of them or just downloading binary because they communicate through the grpc-socket.

Am I understanding correctly?

But your suggestion is quite persuasive to me in perspective of maintenance.

Reply:
Shouldn't we look at and compare Casperlabs' masters when following a version of that source? Only the execution-engine part should be checked for diffs, but there are many other binary and key files. It doesn't matter if you think it doesn't matter.

Reply:
Do you mean to leave the wasm section or the whole .gitignore file as-is?

Reply:
I prefer the whole.

Reply:
But it can also be seen unclear because the .gitignore will contain the elements which not exist in this repository.

May I keep the current contents, if you agree?

Reply:
ND..
How about deleting the mainnet after opening it?

Reply:
Okay~ Then I will keep the .gitignore as-is.

Reply:
df9f9ba as you said